content/guides/tina-cloud/getting-started/markdown-plugin.md

```diff
@@ -27,29 +27,30 @@ yarn add react-tinacms-editor
 
 Inside the `pages/_app.js` file we need to import the `MarkdownFieldPlugin` and add it to the CMS callback:
 
-```jsx
-import '../styles/index.css'
+```diff
 import dynamic from 'next/dynamic'
 import { TinaEditProvider } from 'tinacms/dist/edit-state'
+import "../styles/index.css"
 const TinaCMS = dynamic(() => import('tinacms'), { ssr: false })
-
-// Import the plugin
-import { MarkdownFieldPlugin } from 'react-tinacms-editor'
-
++ import { MarkdownFieldPlugin } from 'react-tinacms-editor'
 const App = ({ Component, pageProps }) => {
   return (
     <>
       <TinaEditProvider
         editMode={
           <TinaCMS
-            // ...
-            // Add it to the cms instance
-            cmsCallback={cms => {
-              cms.plugins.add(MarkdownFieldPlugin)
-            }}
+            clientId={process.env.NEXT_PUBLIC_TINA_CLIENT_ID}
+            branch={process.env.NEXT_PUBLIC_EDIT_BRACH}
+            organization={process.env.NEXT_PUBLIC_ORGANIZATION_NAME}
+            isLocalClient={Boolean(
+              Number(process.env.NEXT_PUBLIC_USE_LOCAL_CLIENT ?? true)
+            )}
++            cmsCallback={cms => {
++              +              cms.plugins.add(MarkdownFieldPlugin)
+                          }}
```

Comment:
Should there be a + on this line as well

Reply:
👎